docs/structured-streaming-programming-guide.md

```diff
@@ -487,11 +487,11 @@ Dataset[Row] csvDF = spark
 spark = SparkSession. ...
 
 # Read text from socket 
-socketDF = spark \
-    .readStream()  \
-    .format("socket") \
-    .option("host", "localhost") \
-    .option("port", 9999) \
+socketDF = spark\
```

Comment:
Using simple or double quote have impact on Python, it's up to the developer's will.

Adding a space before the '\' makes sense, even though I prefere the following syntax for RDD filters:
```
socketDF = (spark
            .readStream()
            .format("socket")
            .option("host", "localhost")
            .option("port", 9999))
```